ui/src/views/network/LbConfigTab.vue

```diff
@@ -0,0 +1,341 @@
+            this.resetValues()
+            this.fetchData()
+          },
+          errorMessage: 'Adding new Load Balancer Config failed',
+          errorMethod: () => {
+            this.resetValues()
+            this.fetchData()
+          },
+          loadingMessage: `Adding new Load Balancer config...`,
+          catchMessage: 'Error encountered while fetching async job result',
+          catchMethod: () => {
+            this.resetValues()
+            this.fetchData()
+          }
+        })
+      }).catch(error => {
+        this.$notifyError(error)
+        this.resetValues()
+        this.fetchData()
+      })
+    },
+    resetValues () {
+      this.lbConfig.name = ''
+      this.lbConfig.description = null
+      this.lbConfig.defaultValue = null
+      this.lbConfig.value = null
+      this.lbConfig.networkid = this.resource.id
+    },
+    populateValues () {
+      for (let i = 0; i < this.globalLbConfigs.length; i++) {
+        if (this.lbConfig.name === this.globalLbConfigs[i].name) {
+          this.lbConfig.description = this.globalLbConfigs[i].description
+          this.lbConfig.defaultvalue = this.globalLbConfigs[i].defaultvalue
+        }
+      }
+    },
+    handleChangePage (page, pageSize) {
+      this.page = page
+      this.pageSize = pageSize
+      this.fetchData()
+    },
+    handleChangePageSize (currentPage, pageSize) {
+      this.page = currentPage
+      this.pageSize = pageSize
+      this.fetchData()
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+.rule {
+  &-container {
+    display: flex;
+    width: 100%;
+    flex-wrap: wrap;
+    margin-right: -20px;
+    margin-bottom: -10px;
+  }
+  &__item {
+    padding-right: 20px;
+    margin-bottom: 20px;
+    @media (min-width: 760px) {
+      flex: 1;
+    }
+  }
+  &__title {
+    font-weight: bold;
+  }
+}
+.add-btn {
+  width: 100%;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  height: auto;
+}
+.add-actions {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: -20px;
+  margin-bottom: 20px;
+  @media (min-width: 760px) {
+    margin-top: 20px;
+  }
+  button {
+    margin-right: 20px;
+  }
+}
+.form {
+  display: flex;
+  margin-right: -20px;
+  margin-bottom: 20px;
+  flex-direction: column;
+  align-items: flex-start;
+  @media (min-width: 760px) {
+    flex-direction: row;
+  }
+  &__item {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    padding-right: 20px;
+    margin-bottom: 20px;
+    @media (min-width: 760px) {
+      margin-bottom: 0;
+      flex: 1;
+    }
+    input,
+    .ant-select {
+      margin-top: auto;
+    }
+  }
+  &__label {
+    font-weight: bold;
+  }
+  &__required {
+    margin-right: 5px;
+    color: red;
+  }
+  .error-text {
+    display: none;
+    color: red;
+    font-size: 0.8rem;
+  }
+  .error {
+    input {
+      border-color: red;
+    }
+    .error-text {
+      display: block;
+    }
+  }
+}
+.pagination {
+  margin-top: 20px;
+}
+</style>
```

Comment:
@weizhouapache Needs a new line at the end of the file
```
 ERROR  Failed to compile with 1 errors10:18:48 AM

 error  in ./src/views/network/LbConfigTab.vue

Module Error (from ./node_modules/eslint-loader/index.js):

/jenkins/workspace/acs-deb-pkg-builder/ui/src/views/network/LbConfigTab.vue
  341:9  error  Newline required at end of file but not found  eol-last

��� 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

Reply:
@davidjumani thanks, it should be fixed now.